ably.d.ts

```diff
@@ -236,8 +236,14 @@ declare namespace Types {
 		ttl?: number;
 	}
 
+	type ChannelParams = Map<string, string>;
```

Comment:
This should be `{ [key: string]: string }` not `Map<string, string>;`.

(I know I wrote `Map<string, string>` in the cross-language spec; that was meant as a 'Map' in the generic sense of a whatever the particular language idiom is for a map from keys to values. It's an unfortunate coincidence that  `Map<string, string>` is valid type specification for Typescript that isn't the one we want in this case, eg since ES6 Maps maps aren't json-serialisable. Apologies for misleading.)

Reply:
Sorry, my mistake - didn't realize that ES6 Map is not JSON serializable.